.github/workflows/links.yaml

```diff
@@ -0,0 +1,48 @@
+# Adapted from UCL ARC's Python Tooling repository
+# https://github.com/UCL-ARC/python-tooling/
+
+# This is a basic workflow to check for broken links with GitHub Actions
+
+name: Link Checker
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'  # Every Sunday at midnight UTC
+
+jobs:
+  link-checker:
+    timeout-minutes: 2
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@885c65f3dc543b57c898c8099f4e08c8afd178a2 # v2.6.1
+        with:
+          # Exclude the image.sc forum and stackoverflow (which both work) but return 403 from
+          # github runners and exclude relative links to other pages. Ignore
```

Comment:
Was there a particular reason to exclude relative links? Can `lychee` not handle these correctly?